interface-definitions/protocols-mpls.xml.in

```diff
@@ -14,6 +14,25 @@
               <help>LDP options</help>
             </properties>
             <children>
+              <node name="parameters">
+                <properties>
+                  <help>MPLS Label Distribution Protocol (LDP) parameters</help>
+                </properties>
+                <children>
+                  <leafNode name="cisco-interop-tlv">
+                    <properties>
+                      <help>Enable Cisco non-compliant format capability TLV</help>
+                      <valueless/>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="transport-prefer-ipv4">
+                    <properties>
+                      <help>Preferred IPv4 for TCP peer transport connection</help>
```

Comment:
This is a bit of a nitpicking, but let's make help string consistent. The unwritten policy is to use verbs, so this should be "Prefer IPv4..."

Reply:
Done